.palantir/revapi.yml

```diff
@@ -774,6 +774,15 @@ acceptedBreaks:
     - code: "java.method.removed"
       old: "method org.apache.iceberg.view.ViewBuilder org.apache.iceberg.view.ViewBuilder::withQueryColumnNames(java.util.List<java.lang.String>)"
       justification: "Acceptable break due to updating View APIs and the View Spec"
+    org.apache.iceberg:iceberg-core:
```

Comment:
we typically don't want to break the API without first going through a deprecation cycle as indicated in https://iceberg.apache.org/contribute/#minor-version-deprecations-required. That being said, if we really want to rename this method, we would have to keep `commitOrClean()` and mark it as deprecated

Reply:
There is some duplicated code when locking in the subclasses.